### ISSUE TITLE
Can't initiate XR 

### ISSUE THREAD

**Issue by matthewhausman** [callstack/react-native-visionos#87]
### Description

Steps in readme don't work to initiate XR experience. XR is undefined on import

### Steps to reproduce

Follow repo instructions

### React Native Version

0.73.0

### Affected Platforms

Runtime - iOS

### Output of `npx react-native info`

```text
System:
  OS: macOS 14.1.2
  CPU: (8) arm64 Apple M2
  Memory: 1.06 GB / 16.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 18.17.0
    path: ~/.nvm/versions/node/v18.17.0/bin/node
  Yarn:
    version: 3.6.4
    path: ~/.nvm/versions/node/v18.17.0/bin/yarn
  npm:
    version: 9.6.7
    path: ~/.nvm/versions/node/v18.17.0/bin/npm
  Watchman: Not Found
Managers:
  CocoaPods: Not Found
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 23.2
      - iOS 17.2
      - macOS 14.2
      - tvOS 17.2
      - visionOS 1.0
      - watchOS 10.2
  Android SDK: Not Found
IDEs:
  Android Studio: Not Found
  Xcode:
    version: 15.2/15C500b
    path: /usr/bin/xcodebuild
Languages:
  Java: Not Found
  Ruby:
    version: 2.7.6
    path: /Users/mhausman/.asdf/shims/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.73.3
    wanted: 0.73.3
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: true
  newArchEnabled: false
```

### Stacktrace or Logs

```text
none
```

### Reproducer

https://github.com/matthewhausman/One-Putt-Golf

**Comment by okwasniewski** [comment]
Hey! Thanks for opening the issue. `XR` API is not yet released (it exists only on `main` branch). I need to do additional tests but I'll try to release it this week

**Comment by mithatb** [comment]
I spend last 3 hours trying to get this worked lol. I wish i read that earlier. :) @okwasniewski

**Comment by okwasniewski** [comment]
Hey, 

`XR` API was released in 0.73.4 👍🏻
